GlobalConfig.json

```
@@ -0,0 +1,10 @@
+{
+  "CompanyName": "MyCompany",
+  "LastModified": "1/26/2021 9:15:47 AM",
+  "ConfigMgrSite": "site.something.com",
+  "ConfigMgrSiteCode": "COM",
+  "RecipeLocations": [
+    "C:\\Temp",
+    "\\\\files.somewhere\\"
```

I think we should add a second item to each location for location type so maybe something like this:
```suggestion
  "RecipeLocations": [
    {
        "locationType": "directory",
        "locationUri": "C:\\Temp"
    },
    {
        "locationType": "directory",
        "locationUri": "\\\\files.somewhere\\"
    }
```

This will make adding additional locations in the future easier (GitHub)

---

Updated the JSON with new arrays for ConfigMgr and RecipeLocations.